src/processBadResponse.js

```diff
@@ -1,33 +1,52 @@
+import { defaultErrorCodes } from './constants';
 import { setAuthError } from './okapiActions';
 
-function getDefaultError() {
-  return {
-    code: 'default.error',
+const getDefaultError = () => (
```

Comment:
Why do we need this functions? I suggest to create constants instead of `getDefaultError` and `getDefaultServerError`?